package.json

```diff
@@ -9,112 +9,35 @@
     },
     "license": "Apache-2.0",
     "author": "Cloudify Platform Ltd. <cosmo-admin@cloudify.co>",
-    "types": "./typings/index.d.ts",
-    "files": [
-        "backend/lib",
-        "backend/typings",
-        "backend/package.json",
-        "backend/*.ts",
-        "cypress",
-        "dist",
-        "configs",
-        "fonts",
-        "images",
-        "scripts",
-        "src",
-        "styles",
-        "typings"
-    ],
-    "main": "dist/cloudify-ui-common.cjs.js",
-    "module": "dist/cloudify-ui-common.esm.js",
-    "browser": "dist/cloudify-ui-common.min.js",
     "scripts": {
-        "audit": "npm audit --production",
-        "build": "npm run build:code && npm run generate-type-declarations && npm run compile-ts",
-        "build:code": "rollup -c",
-        "check-types": "npm run check-types:frontend && npm run check-types:backend",
-        "check-types:frontend": "tsc -p src --noEmit --emitDeclarationOnly false && tsc -p test --noEmit && tsc -p cypress --noEmit",
-        "check-types:backend": "tsc -p backend --noEmit --emitDeclarationOnly false && tsc -p backend/test --noEmit",
-        "compile-ts": "npm run compile-ts:frontend && npm run compile-ts:backend",
-        "compile-ts:frontend": "tsc -p src && tsc -p cypress",
-        "compile-ts:backend": "tsc -p backend",
-        "dev": "rollup -c -w",
-        "generate-type-declarations": "tsc -p src && tsc -p backend",
-        "lint": "eslint --cache --ignore-path .gitignore .",
+        "clean": "git clean -fXd -e \\!node_modules -e \\!node_modules/**/*",
```

Comment:
Useful script to remove all files which are not part of the repository except `node_modules` directory.